tests/test_modeling_flaubert.py

```diff
@@ -399,3 +399,19 @@ def test_model_from_pretrained(self):
         for model_name in FLAUBERT_PRETRAINED_MODEL_ARCHIVE_LIST[:1]:
             model = FlaubertModel.from_pretrained(model_name)
             self.assertIsNotNone(model)
+
```

Comment:
this happens when you create a new branch from a development branch, not from master.